corehq/form_processor/casedb_base.py

```diff
@@ -150,12 +150,19 @@ def get_case_from_case_update(self, case_update, xform):
         """
         case = self.get(case_update.id)
         if case is None:
-            _soft_assert(
-                case_update.creates_case(), "Case created without create block", {
-                    'case_id': case_update.id,
-                    'domain': xform.domain
-                }
-            )
+            if xform.metadata and xform.metadata.commcare_version:
+                from distutils.version import LooseVersion
+                if xform.metadata and xform.metadata.commcare_version:
+                    commcare_version = xform.metadata.commcare_version
+                    _soft_assert(
+                        case_update.creates_case() or commcare_version < LooseVersion("2.35"),
```

Comment:
This asserts that the condition is True, if not it will notify. So this says don't notify if case is being created or version < 2.35.